api/peripheral_api/routes/Speaker.js

```diff
@@ -0,0 +1,80 @@
+const express = require('express');
+const axios = require('axios');
+const router = express.Router();
+const { speakerQueued } = require('../util/Speaker.js');
+const {
+  verifyToken,
+  checkIfTokenSent,
+} = require('../../util/token-verification');
+const {
+  OK,
+  BAD_REQUEST,
+  UNAUTHORIZED,
+  FORBIDDEN,
+  NOT_FOUND,
+  SERVER_ERROR,
+} = require('../../util/constants').STATUS_CODES;
+const logger = require('../../util/logger');
+
+router.post('/stream', async (req, res) => {
+  if (!checkIfTokenSent(req)) {
+    logger.warn('/updateSignText was requested without a token');
+    return res.sendStatus(UNAUTHORIZED);
+  }
+  await axios
+    .post(`http://host.docker.internal:18000/stream/?url=${encodeURIComponent(req.query.url)}`)
```

Comment:
make this a global variable at the top of the file like https://github.com/SCE-Development/Clark/blob/9f0aee7de664b9de37b10b7c0228a2add453f60d/api/peripheral_api/routes/Cleezy.js#L19

the actual url should be passed in by docker-compose.yml like we do for cleezy, see

https://github.com/SCE-Development/Clark/blob/9f0aee7de664b9de37b10b7c0228a2add453f60d/docker-compose.yml#L24